core/src/main/java/io/grpc/CallCredentials.java

```diff
@@ -65,6 +72,7 @@
    * @param applier The outlet of the produced headers. It can be called either before or after this
    *        method returns.
    */
+  @ExperimentalApi("https//github.com/grpc/grpc-java/issues/1914")
   void applyRequestMetadata(
```

Comment:
No, and neither will anyone else.   Please do not make our API support load based on the reading comprehension of our users.

If you want to make this stable and keep experimental methods, please make a base class that extends this.  That is the correct place to put methods that may change.